test/cmdlineTests/ir_unoptimized_with_optimize/args

```diff
@@ -0,0 +1 @@
+--ir --optimize --debug-info none
```

Comment:
Can I have another one with `--ir-optimized --optimize --debug-info none`, so that we can at least see the difference between outputs in this PR?

Reply:
Added. Only for this case though because I don't think such a test is overly useful. `--ir-optimized --optimize` is the standard thing to do and the test is not checking its correctness anyway.